pipe_gaps/core/gap_detector.py

```diff
@@ -48,12 +52,16 @@ def __init__(
         self._normalize_output = normalize_output
 
         self._creation_time = int(datetime.now(tz=timezone.utc).timestamp())
-        self.create_gap = self._create_gap_factory()
 
     @classmethod
     def mandatory_keys(cls):
         """Returns properties that input messages must have."""
-        return [cls.KEY_TIMESTAMP, cls.KEY_DISTANCE_FROM_SHORE]
+        return [
+            cls.KEY_TIMESTAMP,
+            cls.KEY_DISTANCE_FROM_SHORE,
```

Comment:
one question: don't we want to remove it from the previous #23?

Reply:
Yes, but this PR is independent to the other one. Now that I merged the other one, I can rebase this one on top of develop (but could have been the other way around).